sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -1399,12 +1400,12 @@ private void VisitParmVarDecl(ParmVarDecl parmVarDecl, TypedefDecl typedefDecl)
             var parameters = typedefDecl.CursorChildren.Where((cursor) => cursor is ParmVarDecl).Cast<ParmVarDecl>().ToList();
             var index = parameters.IndexOf(parmVarDecl);
             var lastIndex = parameters.Count - 1;
-            
+
             if (name.Equals("param"))
             {
                 _outputBuilder.Write(index);
             }
-            
```

Comment:
👍, the formatter should have removed this and it shouldn't have been committed in the first place